dbapi/src/main/java/com/griddynamics/jagger/dbapi/dto/DecisionPerTestGroupDto.java

```diff
@@ -0,0 +1,52 @@
+package com.griddynamics.jagger.dbapi.dto;
+
+import com.griddynamics.jagger.dbapi.entity.DecisionPerTaskEntity;
+import com.griddynamics.jagger.dbapi.entity.TaskData;
+import com.griddynamics.jagger.util.Decision;
+
+import java.util.List;
+
+public class DecisionPerTestGroupDto {
```

Comment:
according to data structure, TestGroup is also a task => DecisionPerTestGroupDto should extend DecisionPerTaskDto